### PR TITLE
Add Start Building Page to General

### PR DESCRIPTION
**Pull request by filippoweb3**
Explain RFCs and the Polkadot SDK, links to build section and relevant websites

### CODE REVIEW

File: docs/general/start-building.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+id: start-building
+title: Start Building on Polkadot
```
Comment (profradha) [inline docs/general/start-building.md]:
```suggestion
title: Build on Polkadot
```

File: docs/general/start-building.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+sidebar_label: Start Building
```
Comment (profradha) [inline docs/general/start-building.md]:
```suggestion
sidebar_label: Build on Polkadot
```

File: docs/general/start-building.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+description: Start Building with the Polkadot SDK.
+slug: ../start-building
+---
+
+## Request of Commit (RFC)
+
+With the release of [Polkadot runtime 1.0](./polkadot-v1.md), Polkadot's codebase is in the hands of
+the community. Anyone can open a
+[Request For Commit (RFC)](https://github.com/polkadot-fellows/RFCs) to propose and discuss changes
+to network protocol descriptions, runtime logic, and runtime public interfaces,
```
Comment (profradha) [inline docs/general/start-building.md]:
```suggestion
to the network protocol, runtime logic, and public interfaces,
```

File: docs/general/start-building.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+[inherents](../learn/learn-extrinsics.md#types-of-extrinsics), transaction formats, and other
```
Comment (profradha) [inline docs/general/start-building.md]:
```suggestion
 and other
```

File: docs/general/start-building.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+For more information about building on Polkadot, see [the Builder's Guide](../build/build-index.md).
+
+:::
+
+The [Polkadot Software Development Kit (SDK)](https://github.com/paritytech/polkadot-sdk) includes
+all the tools needed to build on the Polkadot ecosystem. The main repositories include:
+
+- Implementation of a node for the Polkadot network in Rust, using the Substrate framework
+- The [Substrate SDK](https://substrate.io/)
+- [Parachain Development Kit (PDK) Cumulus](../build/build-parachains.md#cumulus)
```
Comment (profradha) [inline docs/general/start-building.md]:
```suggestion
- [Cumulus - Parachain Development Kit](../build/build-parachains.md#cumulus)
```

File: docs/general/start-building.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+The used programming language is [Rust](https://www.rust-lang.org/).
```
Comment (profradha) [inline docs/general/start-building.md]:
```suggestion
The programming language used for development is [Rust](https://www.rust-lang.org/).
```

**Review by DrW3RK — APPROVED** [review]
Thanks for adding this! A few suggestions to keep it simple.

**Pull request merged**